reconcile/utils/oc.py

```diff
@@ -957,8 +959,25 @@ class OC:
     def __new__(cls, cluster_name, server, token, jh=None, settings=None,
                 init_projects=False, init_api_resources=False,
                 local=False):
-        use_native = os.environ.get('USE_NATIVE_CLIENT', '').lower() \
-            in ['true', 'yes']
+        use_native = os.environ.get('USE_NATIVE_CLIENT', '')
```

Comment:
current USE_NATIVE_CLIENT will be either "false" or "true", but in both conditions, if use_native will always be true.
```
>>> bool("false")
True
```

Reply:
Fixed